### PR TITLE
Fixed the location of .luacheckrc.

### PR DESCRIPTION
**Pull request by madmaxoft**
Fixes CI builds.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ addons:
 install:
   - luarocks --local install luafilesystem
   - luarocks --local install luacheck
-  - wget -O $TRAVIS_BUILD_DIR/.luacheckrc --no-check-certificate https://builds.cuberite.org/job/Cuberite%20Windows%20x64%20Master/lastSuccessfulBuild/artifact/Server/.luacheckrc
+  - wget -O $TRAVIS_BUILD_DIR/.luacheckrc https://ci.appveyor.com/api/projects/cuberite/cuberite/artifacts/.luacheckrc
```
Comment (mathiascode) [inline .travis.yml:12]:
```suggestion
  - wget -O $TRAVIS_BUILD_DIR/.luacheckrc https://ci.appveyor.com/api/projects/cuberite/cuberite/artifacts/.luacheckrc?job=Windows-x64&pr=false&branch=master
```
We don't want to pull the file from PRs and branches other than master.

**Comment by mathiascode** [comment]
Works now.

**Comment by madmaxoft** [comment]
@bearbin What do you mean? Build? It all works fine with these changes, the file is downloaded from a proper location, so is there any problem that you see?

**Comment by mathiascode** [comment]
The "&" character wasn't interpreted as a part of the URL before, which caused the CI check to fail. I wrapped the URL in quotes to solve the issue.

**Pull request merged**